src/formatters/index.ts

```diff
@@ -1,6 +1,8 @@
 import { trimmer } from 'dtrim';
 import type { LoggerOptions } from 'pino';
 
+export const DEFAULT_MAX_OBJECT_DEPTH = 4;
```

Comment:
Open to suggestions for this default value, given [this](https://github.com/seek-oss/logger/pull/141#discussion_r1680179154) potential usecase.